templatesConfig/noKibana/accounts.go

```diff
@@ -0,0 +1,26 @@
+package noKibana
+
+import (
+	"github.com/ElrondNetwork/elastic-indexer-go/templatesConfig"
+)
+
+type Array = templatesConfig.Array
+type Object = templatesConfig.Object
```

Comment:
bad naming: templatesConfig. Maybe drop the Config part as the package does not hold Configs but hardcoded objects

Reply:
done.